### ISSUE TITLE
vendor/autoload.php missing in Release zip

### ISSUE THREAD

**Issue by t0mm1** [lightszentip/giftlist#32]
**Describe the bug**
When trying to install from Release zip, /public/index.php tries to call /../vendor/autoload.php which doesn't exist.

**To Reproduce**
1. upload contents of zip to new webspace
2. set root to /public/ via .htaccess rewrite rule
3. open in Browser
4. find following error messages:
Warning: require([...]/public/../vendor/autoload.php): failed to open stream: No such file or directory in [...]/public/index.php on line 34 Fatal error: require(): Failed opening required '[...]/public/../vendor/autoload.php' (include_path='.:/usr/share/php:..') in [...]/public/index.php on line 34

**Expected behavior**
Loading of Install Wizard

**Comment by lightszentip** [comment]
Hello @t0mm1 ,

I have download the release file https://github.com/lightszentip/giftlist/releases/download/1.0.0/giftlist-main.zip and the autoload.php exists in this zip file.

Can you give more information about your webserver and the configuation?

For example nginx config for laravel: https://www.digitalocean.com/community/tutorials/how-to-install-and-configure-laravel-with-nginx-on-ubuntu-22-04#step-5-setting-up-nginx

Regards,
lightszentip

**Comment by t0mm1** [comment]
Hi @lightszentip,
thank you for your fast reply. I have a very embarassing update: I thought I rechecked that, but apparently I didn't download the compiled main.zip, I'm very sorry.
I now managed to trace back where I downloaded the files and I must have used the "download zip"-URL that is referenced here, that points to github but to the master branch:
http://lightszentip.github.io/giftlist/

This issue can be closed then, sorry again to bother you and thanks again for the reply. I'll try to get it to run now.
